### PR TITLE
Add IEA to via orthology DAs

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+@Path("/ecoterm")
+@Produces({"application/json"})
+@Consumes({"application/json"})
+public interface EcoTermRESTInterface {
+
+	@GET
+	@Path("/{curie}")
+	@JsonView({View.FieldsAndLists.class})
+	ObjectResponse<ECOTerm> find(@PathParam("curie") String curie);
```
Comment (oblodgett) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java:17]: Does the interface in the curation api not work for this?

Comment (cmpich) [reply agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java:17]: I have not tried but we always had trouble with it.

Comment (oblodgett) [reply agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java:17]: Due to Json view issues but that shouldn't be a problem here.

Comment (oblodgett) [reply agr_indexer/src/main/java/org/alliancegenome/indexer/rest/interfaces/EcoTermRESTInterface.java:17]: We wanted a different view so we had to reimplement the interfaces. However in the end we made the change on the Curation side. So I would guess we don't need the Annotation Serivces / Interfaces anymore either.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**